File: C++/Prawie jak Fibonacci, prawie.cpp
#include <vector>
#include <iostream>

int main() 
{
    int n, count = 0;
    std::cin >> n;
    std::vector<int> sequence(n);
    for (int i = 0; i < n; ++i)
        std::cin >> sequence[i];
    for (int i = 2; i < n; ++i) 
        if (sequence[i] == sequence[i - 1] + sequence[i - 2])
            ++count;
    std::cout << count;
    return 0;
}

File: Java/Prawie jak Fibonacci, prawie.java
import java.util.Scanner;

public class Main
{
    public static void main(String[] args) 
    {
        Scanner input = new Scanner(System.in);
        int n = input.nextInt();
        int[] sequence = new int[n];
        for (int i = 0; i < n; i++)
            sequence[i] = input.nextInt();
        int count = 0;
        for (int i = 2; i < n; i++)
            if (sequence[i] == sequence[i - 1] + sequence[i - 2])
                count++;
        System.out.print(count);
        input.close();
    }
}